misc-utils/uuidd.c

```diff
@@ -55,6 +55,16 @@ struct uuidd_cxt_t {
 			no_sock: 1;
 };
 
+struct uuidd_options_t {
+	const char 	*pidfile_path;
+	const char 	*socket_path;
+	uint32_t	num;
+	int		do_type;
+	unsigned int	do_kill:1,
+			no_pid:1,
+			s_flag:1;
+};
```

Comment:
Please, do it in two steps, 1st add struct uuid_options, 2nd add parse_options(). I don't think we need `_t` suffix as it's always used as `struct` (and no opaque type).